projects/policekillings/database.json

```diff
@@ -0,0 +1,10 @@
+{
+    "prod": {
+      "host": { "ENV" : "DB_HOST" },
+      "user": { "ENV" : "DB_WRITE_USER" },
+      "password" : { "ENV" : "DB_WRITE_PASSWORD" },
+      "database": "milehigh_grassroots_law",
```

Comment:
Recommend moving the database name into an `ENV` constant as well- just one less vulnerability 👍